app/test_main.py

```diff
@@ -1,3 +1,34 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestClass:
+
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, human_age",
+        [
+            (0, 0, [0, 0]),
+            (14, 14, [0, 0]),
+            (15, 15, [1, 1]),
+            (23, 23, [1, 1]),
+            (24, 24, [2, 2]),
+            (27, 27, [2, 2]),
+            (28, 28, [3, 2]),
+            (100, 100, [21, 17]),
+        ]
+    )
+    def test_should_check_everything(
+            self,
+            cat_age: int,
+            dog_age: int,
+            human_age: int
```

Comment:
The type hint for `human_age` should be `List[int]` instead of `int` because `get_human_age` returns a list of integers. Update the type hint to reflect this.